scanamo/src/main/scala/org/scanamo/Table.scala

```diff
@@ -796,6 +802,9 @@ case class Table[V: DynamoFormat](name: String) {
 
   def descending =
     TableWithOptions(name, ScanamoQueryOptions.default).descending
+
+  def transactWriteToTable(vs: List[V]): ScanamoOps[TransactWriteItemsResult] =
```

Comment:
Let's call it `transactPutAll` for now, as I'd like to reserve the `Write` for when we support the full gamut of operations.